v2_bridges/vanilla_apps/peos-on-int/src/main.rs

```diff
@@ -12,11 +12,11 @@ mod usage_info;
 use common::AppError;
 #[cfg(feature = "file-logger")]
 use common_file_logger::init_logger;
-#[cfg(feature = "json-rpc")]
+#[cfg(feature = "json-rpc-db")]
 use common_jsonrpc_db::get_db;
 #[cfg(feature = "rocks-db")]
 use common_rocksdb_database::get_db;
-#[cfg(feature = "stderr-logger")]
+#[cfg(feature = "std-err-logger")]
```

Comment:
uhm why these breaking changes? can we avoid them?

Reply:
I mean the rename of the feature

Reply:
It's unclear to me how exactly that was not spotted by the linter before, but all the way back in 7431b2bbd, that feature gate was introduced without a `stderr-logger` being defined in the relative `Cargo.toml`.
As far as I can tell, there was never a `stderr-logger` feature to begin with, so that portion was effectively broken until now!